.github/workflows/kb_sdk_test.yaml

```diff
@@ -2,40 +2,32 @@ name: KBase SDK Tests
 
 on:
   push:
-    branches: [ master ]
+    branches:
+    - master
+    - main
   pull_request:
-    branches: [ master ]
+    branches:
+    - master
+    - main
+    - develop
 
 jobs:
 
   sdk_tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
I know that GitHub Actions is transitioning `ubuntu-latest` from `22.04` to `24.04`. However, to me the current `ubuntu-latest` is `24.04`, does it make sense to continue using `ubuntu-latest`, or could there be potential issues to consider?

Reply:
Oh, you already approved, Ok, I'll merge and I can change it to 22 if you think that's important